source/dagger/gameplay/team_game/hp_system.cpp

```diff
@@ -0,0 +1,81 @@
+#include "hp_system.h"
+
+#include "core/engine.h"
+#include "core/graphics/sprite.h"
+#include "core/game/transforms.h"
+
+#include <iostream>
```

Comment:
I think you do not use this.

Reply:
You are right, it's a leftover from some testing, will be removed.